discord/ui/select.py

```diff
@@ -260,13 +275,14 @@ def refresh_state(self, interaction: Interaction) -> None:
         self._selected_values = data.get('values', [])
 
     @classmethod
-    def from_component(cls: Type[S], component: SelectMenu) -> S:
+    def from_component(cls: Type[S], select: SelectMenu) -> S:
```

Comment:
No.

A parameter name is part of the contract. This doesn't really matter right now and this is an internal function but you shouldn't just rename parameters.

Reply:
Reverted it. My intention was to make it more consistent. The `discord.ui.Button.from_component` function also uses `button` (not `component `) as parameter name.